swap/cheque.go

```diff
@@ -33,8 +33,8 @@ func (cheque *ChequeParams) encodeForSignature() []byte {
 	// encoded in BigEndian because EVM uses BigEndian encoding
 	binary.BigEndian.PutUint64(cumulativePayoutBytes[24:], cheque.CumulativePayout)
 	// construct the actual cheque
-	input := cheque.Contract.Bytes()
-	input = append(input, cheque.Beneficiary.Bytes()...)
+	input := cheque.ContractAddress.Bytes()
```

Comment:
while i agree that `Contract` is an unacceptable name, i am not a fan of making references to types in variable names.

therefore, my suggestions are: `Chequebook`, `ChequebookContract`

for `Beneficiary`, i would leave it as it is.

as with almost all naming issues, this is up for discussion, so take it with a pinch of salt.